docs/sdk/pubsub/fragments/ios/working-api.md

```diff
@@ -30,7 +30,7 @@ iotDataManager.connectUsingALPN(withClientId: "<YOUR_CLIENT_ID>",
                        statusCallback: mqttEventCallback)
 ```
 
-You can take a look at the [API Reference](https://aws-amplify.github.io/aws-sdk-ios/docs/reference/Classes/AWSIoTDataManager.html#//api/name/connectWithClientId:cleanSession:certificateId:statusCallback:
+You can take a look at the [API Reference](https://aws-amplify.github.io/aws-sdk-ios/docs/reference/AWSIoT/Classes/AWSIoTDataManager.html#/c:objc(cs)AWSIoTDataManager(im)connectUsingALPNWithClientId:cleanSession:certificateId:statusCallback:
 ) to get more information.
```

Comment:
superfluous "⏎"

Reply:
my editor is configured to add a line-break add the end of the files automatically... basically this: https://stackoverflow.com/a/729795

Reply:
now orries. committed. afix.